src/templates/article.js

```diff
@@ -74,6 +76,48 @@ const getSeries = (allSeries, slugTitleMapping, seriesName) => {
     }
 };
 
+const RelatedContainer = styled('div')`
+    background-color: ${colorMap.devBlack};
+    padding: 70px 120px;
+    margin: 0 auto;
+    height: 575px;
+`;
+
+const RelatedCards = styled('div')`
+    display: flex;
+`;
+
+const RelatedArticles = ({ related, slugTitleMapping }) => {
+    if (!related || !related.length) return null;
+    return (
+        <RelatedContainer>
+            <H4>Related</H4>
+            <RelatedCards>
+                {related.map(r => {
+                    const target = r.target;
+                    const slug = r.target && r.target.slice(1, r.target.length);
+                    const image = r.image || ARTICLE_PLACEHOLDER;
+                    const title = dlv(
+                        slugTitleMapping,
+                        [slug, 0, 'value'],
+                        'Title Not Found'
```

Comment:
should this be `''` so we don't output an error message just in case? We can log something instead maybe?